assets/scripts/standard/modals.js

```diff
@@ -0,0 +1,58 @@
+export default function() {
+
+  var modalTriggers = document.querySelectorAll('.js-modal-trigger');
+  var modalTriggersArr = [].slice.call(modalTriggers);
+  var modalTriggersLen = modalTriggers && modalTriggers.length || 0;
+  var closeBtns = document.querySelectorAll('.js-modal-close-btn');
+  var closeBtnsLen = closeBtns && closeBtns.length || 0;
+  var overlay = document.querySelector('.js-modal-overlay');
+  var activeModal;
+  var transitionClass;
+  var overlayClass;
+  var i;
+
+
+  function closeModal() {
+    activeModal.classList.remove(transitionClass);
+    overlay.classList.remove(overlayClass);
+    overlay.style.visibility = 'hidden';
+  }
+
+  function addOverlayHandler() {
+    overlay.addEventListener('click', closeModal);
```

Comment:
@Jing-Ran 
I had to change this line to prevent JS error when the modals are not present on the page:
`overlay.addEventListener('click', closeModal);` this is what I did:
`overlay && overlay.addEventListener('click', closeModal);`

If you want, you can change it and add a different condition.